src/System.Collections.Specialized/System.Collections.Specialized.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
-VisualStudioVersion = 12.0.30723.0
+# Visual Studio 14
```

Comment:
I've been undoing these changes that VS makes automatically.  But it's probably not a big deal.